### PR TITLE
devops: Swap `Makefile` for `Taskfile`

### PR DESCRIPTION
**Pull request by max-sixty**
Mostly the work of an LLM — pretty good at this sort of thing...

### CODE REVIEW

**Comment by richb-hanover** [comment]
@max-sixty I'm curious - what did you ask the LLM to do?

File: prqlc/bindings/prqlc-c/examples/minimal-zig/Taskfile.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+version: 3
+
+vars:
+  project_root: "../../../../.."
+
+tasks:
+  run:
+    cmds:
+      - task: build
```
Comment (eitsupi) [inline prqlc/bindings/prqlc-c/examples/minimal-zig/Taskfile.yml:11]: I suppose this should be `deps`

File: prqlc/bindings/prqlc-c/examples/minimal-zig/Taskfile.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    cmds:
+      - cargo build --package prqlc-c --release
+      - mkdir -p c/
+      - cp {{.project_root}}/prqlc/bindings/prqlc-c/prqlc.h c/
+      - cp {{.project_root}}/target/release/libprqlc_c.* c/
+
+  build:
+    desc: "Build the project"
+    cmds:
+      - zig build
```
Comment (eitsupi) [inline prqlc/bindings/prqlc-c/examples/minimal-zig/Taskfile.yml:26]: Is `source` and `generated` unnecessary?

**Comment by max-sixty** [comment]
> @max-sixty I'm curious - what did you ask the LLM to do?

Just rewrite this Makefile as a Taskfile... (I then edited it a bit)

File: prqlc/bindings/prqlc-c/examples/minimal-zig/Taskfile.yml (hunk 2 above)
Comment (max-sixty) [reply prqlc/bindings/prqlc-c/examples/minimal-zig/Taskfile.yml:26]: It's not necessary, though it can help with caching

**Comment by richb-hanover** [comment]
> Just rewrite this Makefile as a Taskfile...

Hmmm... Not much in the way of instruction/direction. That is so delightful. Better than a sophomore CS major :-)

**Pull request merged**